roads.mss

```diff
@@ -2830,6 +2835,7 @@ tertiary is rendered from z10 and is not included in osm_planet_roads. */
       text-clip: false;
       text-placement: line;
       text-face-name: @book-fonts;
+      text-repeat-distance: @text-repeat-distance;
```

Comment:
This code is not working properly in my tests. It decreases amount of text repetitions, but does it in the same way for all values, no matter what global value is set. However if you make local value, like `text-repeat-distance: 50;`, it works different way for different values, as it should be. See https://github.com/gravitystorm/openstreetmap-carto/pull/3295#issuecomment-404984750.

Proposed solutions:
- Investigate why global values works as a uniform "small repetitions" switch, while local values work as expected.
- Make all these values local (and test that they all work as expected).